packages/showcases/src/pages/NoSSR/pdf.tsx

```diff
@@ -7,21 +7,39 @@ import JinaClient, {
     SimpleQueries,
     AnyObject,
     SimpleResponse,
-    SimpleResult
+    SimpleResult,
+    fileToBase64
 } from "@jina-ai/jinajs";
 import React, {useState} from "react";
 import Results from "../../components/Results";
+import downloadButton from '../../images/download-button.svg'
```

Comment:
These shouldn't be images. Please create buttons. It's better for accessibility.